Helper/Data.php

```diff
@@ -1388,9 +1421,19 @@ public function getOpenInvoiceLineData(
     public function getPosApiKey($storeId = null)
     {
         if ($this->isDemoMode($storeId)) {
-            $apiKey = $this->_encryptor->decrypt(trim($this->getAdyenPosCloudConfigData('api_key_test', $storeId)));
+            $encryptedApiKeyTest = $this->getAdyenPosCloudConfigData('api_key_test', $storeId);
+            if (is_null($encryptedApiKeyTest)) {
+                return false;
+            }
+
+            $apiKey = $this->_encryptor->decrypt(trim($encryptedApiKeyTest));
         } else {
-            $apiKey = $this->_encryptor->decrypt(trim($this->getAdyenPosCloudConfigData('api_key_live', $storeId)));
+            $encryptedApiKeyLive = $this->getAdyenPosCloudConfigData('api_key_live', $storeId);
+            if (is_null($encryptedApiKeyLive)) {
+                return false;
```

Comment:
Let's turn `string` configurations into `string|null` to represent lack of setup:
```suggestion
                return null;
```